docs/general/information-schema.rst

```diff
@@ -294,11 +294,11 @@ table::
     | locations         | description                    |   6 | text                     |
     | locations         | id                             |   1 | integer                  |
     | locations         | information                    |  11 | object_array             |
-    | locations         | information['evolution_level'] |  12 | smallint                 |
-    | locations         | information['population']      |  13 | bigint                   |
+    | locations         | information['evolution_level'] |  13 | smallint                 |
+    | locations         | information['population']      |  12 | bigint                   |
     | locations         | inhabitants                    |   7 | object                   |
-    | locations         | inhabitants['description']     |   8 | text                     |
-    | locations         | inhabitants['interests']       |   9 | text_array               |
+    | locations         | inhabitants['description']     |   9 | text                     |
+    | locations         | inhabitants['interests']       |   8 | text_array               |
```

Comment:
The positions for inner object types were not in the order of how the CREATE TABLE statement declared them, see https://github.com/crate/crate/blob/master/blackbox/test_docs.py#L182.